estate/models/estate_property.py

```diff
@@ -0,0 +1,137 @@
+from datetime import datetime
+from odoo import models,fields,api,_
+from dateutil.relativedelta import relativedelta
+from odoo.exceptions import UserError
```

Comment:
```suggestion
from datetime import datetime
from dateutil.relativedelta import relativedelta

from odoo import api, fields, models
from odoo.exceptions import UserError
```

Generally we first make external imports and then we import from odoo in alphabetical order.
Need to adapt in all files